scripts/fastq-to-fasta.py

```diff
@@ -67,7 +70,7 @@ def main():
     else:
         print('No lines dropped from file.', file=sys.stderr)
 
-    print('Wrote output to', args.output, file=sys.stderr)
+    print('Wrote output to', str(args.output), file=sys.stderr)
```

Comment:
Fails if we pass sys.stdout as output--it doesn't have a name.

That said, my `str` solution doesn't really help. I should check to see if it is in fact a block device or whathaveyou.

@mr-c reccomended making a function to do checking on file handles to see if they are fifos, block devices, etc. and factoring those checks out of the rest of the codebase.